molecule/default/tests/test_default.py

```diff
@@ -1,20 +1,60 @@
 import os
 import re
-
+import pytest
 import testinfra.utils.ansible_runner
 
+###############################################################
+# ASC-187: Verify nova_force_config_drive setting is disabled  #
+################################################################
+# RPC 10+ manual test 1
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('compute-infra_hosts')
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts(
+    ['nova_scheduler', 'nova_compute'])
```

Comment:
nit pick: for linting purposes, I believe this line should be indented.

Reply:
I ran `pycodestyle` against the file and it passed PEP8, not sure if the indentation is needed